include/libtorrent/ssl.hpp

```diff
@@ -111,7 +115,7 @@ using boost::asio::gnutls::context;
 using boost::asio::gnutls::stream_base;
 using boost::asio::gnutls::stream;
 using boost::asio::gnutls::verify_context;
-using boost::asio::gnutls::rfc2818_verification;
+using boost::asio::gnutls::host_name_verification;
```

Comment:
Does this work with older versions of boost?

Reply:
no, but master requires boost that isn't too old. I don't know the cut-off off hand

Reply:
actually, this specific line isn't boost, it's the gnutls socket wrappers, and they do support `host_name_verification`